publiccode.yaml

```diff
@@ -1,51 +1,104 @@
 description:
   en:
-    apiDocumentation: ''
-    awards: []
-    documentation: ''
-    features: []
+    localisedName: null
     genericName: A bundle to run the Notification Component (NRC)
-    localisedName: ''
-    longDescription: ''
-    screenshots: []
     shortDescription: NotificiationsBundle
+    longDescription: A bundle to run the Notification Component (NRC)
+    documentation: null
+    apiDocumentation: null
+    features: []
+    screenshots: []
     videos: []
+    awards: []
   nl:
+    localisedName: null
+    genericName: Een bundel om het Notification Component (NRC) te kunnen gebruiken
     shortDescription: NotificiationsBundle
-developmentStatus: beta
-landingURL: ''
+    longDescription: Een bundel om het Notification Component (NRC) te kunnen gebruiken
+    documentation: null
+    apiDocumentation: null
+    features: []
+    screenshots: []
+    videos: []
+    awards: []
+intendedaudience:
+  countries:
+    - nl
+  unsupportedCountries: []
+  scope:
+    - government
+    - local-authorities
 legal:
-  authorsFile: ''
-  license: ''
-  mainCopyrightOwner: ''
-  repoOwner: ''
+  license: EUPL-1.2-or-later
+  mainCopyrightOwner: legal/authorsFile
+  repoOwner: CommonGateway
+  authorsFile: null
+maintenance:
+  type: internal
+  contractors:
+    - name: Conduction
+      until: 2035-01-01
+      email: info\x64conduction.nl
+      website: https://www.conduction.nl
+  contacts:
+    - name: Ruben van der Linde
+      email: info\x64conduction.nl
+      phone: 085 303 6840
+      affiliation: Conduction
 localisation:
+  localisationReady: true
   availableLanguages:
-  - en
-  localisationReady: false
-maintenance:
-  contacts: []
-  contractors: []
-  type: none
-name: NotificiationsBundle
+    - en
+    - nl
+dependsOn:
+  open:
+    - name: CommonGateway
+      versionMin: 1.2
+      versionMax: 1.3
+      version: 1.2.47
+      optional: false
+    - name: CoreBundle
+      versionMin: 1.2
+      versionMax: 1.3
+      version: 1.2.47
+      optional: false
+roadmap: null
+inputTypes:
+  - application/json
+outputTypes:
+  - application/json
+  - application/json+hal
+  - application/json+ld
+  - application/xml
+  - application/x-yaml
 nl:
-  vng:
-    commonground: []
-    gemma: []
-organisation:
-  description: The common gateway
-  logo: https://avatars.githubusercontent.com/u/106860777?v=4
-  name: CommonGateway
-  url: CommonGateway
-platforms:
-- web
-publiccodeYmlVersion: '0.1'
-releaseDate: '2022-10-01'
-roadmap: ''
-softwareType: addon
-softwareVersion: ''
-url: https://github.com/CommonGateway/NotificiationsBundle
-usedBy: []
+  countryExtensionVersion: null
+  commonground:
+    intendedOrganisations: []
+    installationType: null
+    layerType: service
+  gemma:
+    bedrijfsfuncties: []
+    bedrijfsservices: []
+    applicatiefunctie: null
+    model: null
+  upl: []
+downloadUrls:
+  - https://github.com/CommonGateway/NotificiationsBundle/archive/refs/heads/main.zip
```

Comment:
should be https://github.com/CommonGateway/NotificiationsZGWBundle/archive/refs/heads/main.zip